src/views/app/detail-panel/preview/parts/preview-header.tsx

```diff
@@ -116,11 +120,11 @@ const PreviewHeader: FC<PreviewHeaderProps> = ({ compProps }): ReactElement => {
 		() =>
 			reduce(
 				tagsFromStore,
-				(acc: any, v) => {
+				(acc: Array<Tag & { label: string; customComponent: ReactElement }>, v) => {
 					if (includes(message.tags, v.id))
 						acc.push({
 							...v,
-							color: ZIMBRA_STANDARD_COLORS[v.color ?? 0].hex,
+							color: parseInt(ZIMBRA_STANDARD_COLORS[v.color ?? 0].hex, 10),
```

Comment:
Did you check if the value passed it is right? I don't think a hex could be properly converted into number